Feat/pnl updates

matthew-garrett opened the pull request:
- Fix analytics table style bug
- Adds leverage points icon to dropdown
- Updates pnl data tables
- Fixes sNOTE holding bugs
- Fixes portfolio pending message spacing issues
- Replaces 0 with dashes

jeffywu commented on packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx:
```diff
@@ -140,11 +152,11 @@ export function useEarningsBreakdown(isGrouped: boolean) {
+          totalIncentiveEarnings &&
+          totalIncentiveEarnings.toFloat() > minEarningsValue
+            ? formatCryptoWithFiat(baseCurrency, totalIncentiveEarnings)
+            : '-',
```
I thought we were going to just show a zero in these cases? I don't think we should be doing this minEarningsValue comparison anywhere

jeffywu commented on packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx:
```diff
@@ -56,6 +60,7 @@ export function useEarningsBreakdown(isGrouped: boolean) {
   const baseCurrency = useFiat();
   const holdings = usePortfolioHoldings(network);
   const groupedHoldings = useGroupedHoldings(network) || [];
+  const minEarningsValue = 0.01;
```
I think this should be removed altogether

jeffywu commented on packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx:
```diff
@@ -201,11 +248,25 @@ export function useEarningsBreakdown(isGrouped: boolean) {
+                    ? totalIncentiveEarnings
+                        ?.toFiat(baseCurrency)
+                        .toDisplayStringWithSymbol(2)
+                    : '-',
```
We should show zero on null

matthew-garrett replied:
Teddy made a ticket to always show a dash https://app.asana.com/0/1205751432906245/1207540343338899

Lmk if you want me to change it back to 0

jeffywu commented on packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx:
```diff
@@ -238,6 +238,29 @@ export function useEarningsBreakdown(isGrouped: boolean) {
+              ? totalIncentiveEarnings
+                  ?.toFiat(baseCurrency)
+                  .toDisplayStringWithSymbol(2)
+              : `${FiatSymbols[baseCurrency]} 0.00`
```
Is there supposed to be a space between the symbol and the zeros?

matthew-garrett committed:
build: bugfixes and pnl updates wip

build: add pnl table changes

build: removes console log

build: removes minEarningsValue usage

build: updates zeros and dashes in earnings breakdown table

build: addresses comment
